.mergify.yml

```diff
@@ -96,10 +96,18 @@ pull_request_rules:
         add:
           - new CVE
 merge_protections:
-  - name: changelog requirements
+  - name: Changelog requirements
     if: []
     success_conditions:
       - or:
           - '-title ~= ^feat'
           - label = skip changelog
           - label = need changelog
+  - name: Enforce conventional commit
```

Comment:
There's a post_check action above that does the same thing, shouldn't we remove it as well ?

Reply:
Yes, I'm doing that in two steps to make sure it works well.